build/workflow/controller/Dockerfile

```diff
@@ -6,4 +6,4 @@ WORKDIR /workspace
 
 COPY ./bin/workflow/controller /workspace/controller
 
-CMD ["./controller"]
\ No newline at end of file
+ENTRYPOINT ["./controller"]
```

Comment:
Why use ENTRYPOINT ?

Reply:
Need to pass `--namespace` flag. Refer to  https://github.com/caicloud/cyclone/pull/981/files#diff-f31db4d87194712918aaca8ebba3a978R82

Reply:
This can be removed

Reply:
Fixed